chancloser.go

```diff
@@ -438,33 +432,16 @@ func (c *channelCloser) ProcessCloseMsg(msg lnwire.Message) ([]lnwire.Message, b
 		if err := c.cfg.broadcastTx(closeTx); err != nil {
 			return nil, false, err
 		}
-
-		// Clear out the current channel state, marking the channel as
-		// being closed within the database.
-		closingTxid := closeTx.TxHash()
-		chanInfo := c.cfg.channel.StateSnapshot()
-		c.closeCtx.Finalize(&channeldb.ChannelCloseSummary{
-			ChanPoint:      c.chanPoint,
-			ChainHash:      chanInfo.ChainHash,
-			ClosingTXID:    closingTxid,
-			CloseHeight:    c.negotiationHeight,
-			RemotePub:      &chanInfo.RemoteIdentity,
-			Capacity:       chanInfo.Capacity,
-			SettledBalance: finalLocalBalance,
-			CloseType:      channeldb.CooperativeClose,
-			ShortChanID:    c.cfg.channel.ShortChanID(),
-			IsPending:      true,
-		})
-
-		// TODO(roasbeef): don't need, ChainWatcher will handle
-
-		c.state = closeFinished
+		if c.cfg.channel.MarkCommitmentBroadcasted(); err != nil {
```

Comment:
The one side-effect of doing this, is that for a co-op close, we won't show what the proper _final_ balances are (to account for the negotiated fees) until the closing transaction is fully closed.